diff --git a/dist/cjs/util.js b/dist/cjs/util.js
index 77e64b56872bc65bcb0f60ecfbec84eb3da02c6f..8d5b81eea636c3f1672cfe2c66602078bda1e847 100644
--- a/dist/cjs/util.js
+++ b/dist/cjs/util.js
@@ -153,7 +153,7 @@ async function walkAsync(dirPath) {
         const children = await fs.readdir(dirPath);
         return await Promise.all(children.map(async (child) => {
             const filePath = path.resolve(dirPath, child);
-            const stat = await fs.stat(filePath);
+            const stat = await fs.lstat(filePath);
             if (stat.isFile()) {
                 switch (path.extname(filePath)) {
                     case '.cstemp': // Temporary file generated from past codesign
diff --git a/dist/esm/util.js b/dist/esm/util.js
index d3f66d54b1a88e7410d78affe17de87302db02a3..e86b066d1d21628908ecc848f5bb29a7598605a7 100644
--- a/dist/esm/util.js
+++ b/dist/esm/util.js
@@ -117,7 +117,7 @@ export async function walkAsync(dirPath) {
         const children = await fs.readdir(dirPath);
         return await Promise.all(children.map(async (child) => {
             const filePath = path.resolve(dirPath, child);
-            const stat = await fs.stat(filePath);
+            const stat = await fs.lstat(filePath);
             if (stat.isFile()) {
                 switch (path.extname(filePath)) {
                     case '.cstemp': // Temporary file generated from past codesign
